pytab/worker.py

```diff
@@ -115,15 +115,15 @@ def workMan(worker_count, ffmpeg_cmd):
                 frames.append(int(float(new_line[0].split("=")[-1])))
                 framerates += int(float(new_line[1].split("=")[-1]))
                 speeds.append(float(new_line[6].split("=")[-1].replace("x", "")))
-            lineAmmount = len(framelines)
-            if lineAmmount == 0:
-                lineAmmount = 1
+            lineAmount = len(framelines)
```

Comment:
ignoring the obvious variable not in snake_case

Reply:
Well... 
Agreed! 
_Oopsies..._

Reply:
you agree to lineAmount or to line_amount?

Reply:
LineAmount is fine... otherwise a lot of variables would need renaming too.